### PR TITLE
Add sso token exchange skill handler package

### PR DESCRIPTION
**Pull request by EricDahlvang**

### CODE REVIEW

File: libraries/Bot.Builder.Community.Components.TokenExchangeSkillHandler/SkillsConfiguration.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+using Microsoft.Extensions.Configuration;
+
+namespace Bot.Builder.Community.Components.TokenExchangeSkillHandler
+{
+    internal class SkillsConfiguration
+    {
+        public SkillsConfiguration(IConfiguration configuration)
+        {
+            var section = configuration?.GetSection("skill");
+            foreach (var child in section.GetChildren())
```
Comment (rvinothrajendran) [inline libraries/Bot.Builder.Community.Components.TokenExchangeSkillHandler/SkillsConfiguration.cs:14]: Null condition has to check section object ?

File: libraries/Bot.Builder.Community.Components.TokenExchangeSkillHandler/SkillsConfiguration.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+            foreach (var child in skills)
+            {
+                var props = child.GetChildren().ToDictionary(x => x.Key, x => x.Value);
+
+                Skills.Add(child.Key, new BotFrameworkSkill()
+                {
+                    Id = child.Key,
+                    AppId = props["msAppId"],
+                    SkillEndpoint = new Uri(props["endpointUrl"]),
```
Comment (rvinothrajendran) [inline libraries/Bot.Builder.Community.Components.TokenExchangeSkillHandler/SkillsConfiguration.cs:28]: @EricDahlvang  If the keys(appsettings msAppId & endpointUrl ) is missing in appsettings file , throw InvalidOperationException ?

Comment (EricDahlvang) [reply libraries/Bot.Builder.Community.Components.TokenExchangeSkillHandler/SkillsConfiguration.cs]: Good point. Fixed, added some null ref check tests

**Review by rvinothrajendran — COMMENTED** [review]
Review has done

**Pull request merged**